### PR TITLE
Add setting doc for auto disk provisioning and ssl parameter

### PR DESCRIPTION
**Pull request by johnliu55tw**
- Add systems settings `auto-disk-provision-paths` and `ssl-parameters`.
- Reference the system settings page in installer config `system_settings`.
- Add troubleshooting guide for resetting `ssl-parameters`
- Fix several trailing spaces.

### CODE REVIEW

File: docs/settings/settings.md [hunk 1]
```diff
@@ -175,3 +175,49 @@ Default: none
...
+
+!!! warning
+    - This setting is applied to **every Node** in the cluster.
+    - All the data in these storage devices **will be destroyed**. Use at your own risk.
+
+Default: none
+
+#### Example
+
+Add disks matches glob pattern `/dev/sd*` or `/dev/hd*`:
```
Comment (nunix) [inline docs/settings/settings.md:192]:
```suggestion
The following example, will add disks matching the glob pattern `/dev/sd*` or `/dev/hd*`:
```

File: docs/settings/settings.md [hunk 2]
```diff
@@ -175,3 +175,49 @@ Default: none
...
+This setting allows you to change the enabled SSL/TLS protocols and ciphers of Harvester GUI and API.
+
+The following options and values can be set:
+
+- `protocols`: Enabled protocols. See NGINX Ingress Controller's configs [`ssl-protocols`](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/#ssl-protocols) for supported input.
+
+- `ciphers`: Enabled ciphers. See NGINX Ingress Controller's configs [`ssl-ciphers`](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/#ssl-ciphers) for supported input.
+
+If no value is provided, default protocols are set to `TLSv1.2` only and ciphers are
+`ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305`.
```
Comment (nunix) [inline docs/settings/settings.md:209]:
```suggestion
If no value is provided, `protocols` is set to `TLSv1.2` only and the `ciphers` list is
`ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305`.
```

Comment (nunix) [reply docs/settings/settings.md:209]: for the wording `cyphers list`, I checked the NGINX page referenced

File: docs/settings/settings.md [hunk 3]
```diff
@@ -175,3 +175,49 @@ Default: none
...
+
+Default: none
+
+#### Example
+
+Set SSL/TLS enabled protocols to `TLSv1.2` and `TLSv1.3` and enabled ciphers to
+`ECDHE-ECDSA-AES128-GCM-SHA256` and `ECDHE-ECDSA-CHACHA20-POLY1305`.
```
Comment (nunix) [inline docs/settings/settings.md:216]:
```suggestion
The following example sets the enabled SSL/TLS protocols to `TLSv1.2` and `TLSv1.3` and the ciphers list to
`ECDHE-ECDSA-AES128-GCM-SHA256` and `ECDHE-ECDSA-CHACHA20-POLY1305`.
```

**Review by nunix — COMMENTED** [review]
Rephrased the examples introduction and the wording for `cyphers` based on NGINX referenced page.

File: docs/settings/settings.md [hunk 4]
```diff
@@ -175,3 +175,52 @@ Default: none
 ```
 ens3
 ```
+
+## `auto-disk-provision-paths`
+
+This setting allows Harvester to automatically add disks that match the given glob pattern as VM storages.
```
Comment (guangbochen) [inline docs/settings/settings.md:181]: \s\storages\storage

File: docs/settings/settings.md [hunk 5]
```diff
@@ -175,3 +175,52 @@ Default: none
...
+It's possible to provide multiple patterns by separating them with comma.
```
Comment (guangbochen) [inline docs/settings/settings.md:182]: \s\comma\a comma

File: docs/settings/settings.md [hunk 6]
```diff
@@ -175,3 +175,52 @@ Default: none
...
+
+!!! warning
+    - This setting is applied to **every Node** in the cluster.
+    - All the data in these storage devices **will be destroyed**. Use at your own risk.
+
+Default: none
+
+#### Example
+
+The following example, will add disks matching the glob pattern `/dev/sd*` or `/dev/hd*`:
```
Comment (guangbochen) [inline docs/settings/settings.md:192]: remove the `,` after example

File: docs/troubleshooting/harvester.md [hunk 7]
```diff
@@ -26,3 +26,37 @@ You can access the embedded Longhorn UI via `https://{{HARVESTER_IP}}/dashboard/
...
+```
+apiVersion: harvesterhci.io/v1beta1
+default: '{}'
+kind: Setting
+metadata:
+  name: ssl-parameters
+...
+value: '{"protocols":"TLS99","ciphers":"WRONG_CIPHER"}' # <- Delete this line
+```
+4. Save the change and you should see the following response after exited from the editor:
```
Comment (guangbochen) [inline docs/troubleshooting/harvester.md:57]: \s\exited\exit

**Review by guangbochen — COMMENTED** [review]
also can we please help to resolve the conflict, thanks?

**Review by guangbochen — APPROVED** [review]
LGTM

**Pull request merged**